config.py

```diff
@@ -3,41 +3,50 @@
 import os.path
 import logging
 import math
-from matchmaking import DelayType
+from abc import ABCMeta
+from enum import Enum
+from typing import Dict, Any
+CONFIG_DICT_TYPE = Dict[str, Any]
 
 logger = logging.getLogger(__name__)
 
 
+class DelayType(str, Enum):
```

Comment:
Why was `DelayType` copied here?

Reply:
Because there was a circular import. This is why `MAX_CHAT_MESSAGE_LEN` was also copied. Because I added `from config import Configuration` in matchmaking.py I couldn't import anything from matchmaking.

Reply:
For `MAX_CHAT_MESSAGE_LEN`, you can delete the definition in engine_wrapper.py and use `lichess.MAX_CHAT_MESSAGE_LEN` inside the `get_stats()` method.

Similarly for `DelayType`, delete the definition in matchmaking.py and use `config.DelayType` instead.

This removes duplicate definitions.